### DISCUSSION TITLE
MessageHandlers,Filters Are not working In Replit

### DISCUSSION THREAD

**Discussion by Diwanshujii** [python-telegram-bot/python-telegram-bot#3473] [Q&A]
I'm trying to use the MessageHandler class in my Telegram bot, but I keep getting an error that says 'NameError: name 'MessageHandler' is not defined'. I've imported the telegram library and I don't see any other issues with my code. What could be causing this error and how can I fix it?

**Comment by harshil21** [comment]
What PTB version are you on? In v20, it's `from telegram.ext import MessageHandler, filters`
